### PR TITLE
Nylon Scabbard MOLLE Straps

### PR DESCRIPTION
**Pull request by Kantonine**
Nylon Scabbard MOLLE straps

#### Summary
Balance "Nylon Scabbard can attach to MOLLE panels"

#### Purpose of change
Nylon Scabbards weren't all that much different than their leather counterparts. Wanted to differentiate them more, and give them extra versatility

#### Describe the solution
Gave nylon scabbards the PALS_LARGE flag, just like Nylon Quivers, and Large Holsters

### CODE REVIEW

**Comment by Hyperseeker** [comment]
Do you have any particular source which would justify this? I'm struggling to find anything even remotely agreeing with the concept of carrying a sword using PALS webbing. The closest thing I'm able to find is either

* a sheath for knives
* a rifle/shotgun scabbard
* [this thing](https://www.reliks.com/belts-frogs-and-scabbards/honshu-molle-attachment/), which accepts a specific type of a scabbard itself
* [this thing](https://www.swordnarmory.com/new/12-sword-frog-back-scabbard-sheath-black-nylon-tactical-modular-camp-cosplay/), similar in idea, but instead of being MOLLE-compatible, it provides a lot of PALS of its own

([This type of a sheath](https://apocsurvivaltools.com/products/atrim-survival-tanto) could, in theory, mount onto PALS webbing with the right adapter, but it doesn't natively.)

I suspect the issue here, just like with nylon quivers, is a misunderstanding. If anything, a nylon scabbard would provide its own webbing to attach pouches onto, just like every single "MOLLE quiver" I've seen. This webbing *could* be used to attach it to another webbing with the right set of straps, but again, I feel like this has to be recognized in some way.

(As an aside, the large holster – or rather, the item it's likely modelled after – has nothing to do with MOLLE pouches and is, instead, an independent leg-mounted holster.)

File: data/json/items/armor/sheath.json [hunk 1]
```diff
@@ -429,7 +429,7 @@
     "id": "nylon_scabbard",
     "type": "ARMOR",
     "name": { "str": "nylon scabbard" },
-    "description": "A large, adjustable sheath for holding swords and other large blades, made with nylon.  Activate to sheathe/draw a weapon.",
+    "description": "A large, adjustable, MOLLE compitable sheath for holding swords and other large blades, made with nylon.  Activate to sheathe/draw a weapon.",
```
Comment (DSeyka) [inline data/json/items/armor/sheath.json:432]:
```suggestion
    "description": "A large, adjustable, MOLLE compatible sheath for holding swords and other large blades, made with nylon.  Activate to sheathe/draw a weapon.",
```

**Comment by IdleSol** [comment]
How do you envision it? A sword scabbard (~120 cm) attached to an armored vest (for example).

**Comment by Maleclypse** [comment]
I was trying to find examples of this in the wild and came across this interesting thing.  https://voodootactical.com/new-items/vtc-voodoo-tactical-camo/shotgun-scabbard/

**Comment by Maleclypse** [comment]
So what I'm actually finding is that there is a molle attachment that will go around scabbards but not a scabbard with a built in molle attachment.  Can you please share some examples of this item or change it to a molle attaching pocket that holds a scabbard?

**Pull request closed without merging**